react-backend/client/src/cb-admin/pages/Signup.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+// import { pick, pathOr, equals } from 'ramda';
+// import { CbAdmin } from '../../api';
+import { Form, FormSection, PrimaryButton } from '../../shared/components/form/base';
+import { Heading, Paragraph, Link } from '../../shared/components/text/base';
+import LabelledInput from '../../shared/components/form/LabelledInput';
+import LabelledSelect from '../../shared/components/form/LabelledSelect';
+import { FlexContainerCol } from '../../shared/components/layout/base';
+
+const categories = [
+  { key: '1', value: 'Art centre or facility' },
+  { key: '2', value: 'Community hub, facility or space' },
+  { key: '3', value: 'Community pub, shop or café' },
+  { key: '4', value: 'Employment, training, business support or education' },
+  { key: '5', value: 'Energy' },
+  { key: '6', value: 'Environment or nature' },
+  { key: '7', value: 'Food catering or production (incl. farming)' },
+  { key: '8', value: 'Health, care or wellbeing' },
+  { key: '9', value: 'Housing' },
+  { key: '10', value: 'Income or financial inclusion' },
+  { key: '11', value: 'Sport & leisure' },
+  { key: '12', value: 'Transport' },
+  { key: '13', value: 'Visitor facilities or tourism' },
+  { key: '14', value: 'Waste reduction, reuse or recycling' },
+];
+
+const regions = [
+  { key: '1', value: 'North East' },
+  { key: '2', value: 'North West' },
+  { key: '3', value: 'Midlands' },
+  { key: '4', value: 'Wales' },
+  { key: '5', value: 'South West' },
+  { key: '6', value: 'South East' },
+  { key: '7', value: 'Greater London' },
+];
+
+// const payloadFromState = pick([
+//   'org_name',
+//   'category',
+//   'email',
+//   'password',
+//   'password_confirm',
+// ]);
+
+// const getErrorStatus = pathOr(null, ['response', 'status']);
```

Comment:
This pathOr means that if the data we're looking for is not there `response.status` it will return whatever default you've set, in this case `null` right?

Reply:
Correct

Reply:
Ooo and curried...